src/View/Window.vala

```diff
@@ -892,31 +891,32 @@ namespace Marlin.View {
 
         private void save_tabs () {
             VariantBuilder vb = new VariantBuilder (new VariantType ("a(uss)"));
```

Comment:
Micro-optimization: this variable can go into the `if` block.

Since the whole method is a no-op without the `remember_history` setting, an early-exit `return` would keep the indentation flat, as in the `restore_tabs` method.